app/journey/Journey.scala

```diff
@@ -94,21 +93,18 @@ final case class Journey(
     require(eligibilityStatus.eligible, s"taxpayer has to be eligible [$this]")
   }
 
-  // TODO OPS-9464: change to when old journey removed '.getOrElse(throw new IllegalArgumentException("attempted to retrieve total spending when there was no spending"))`
   def remainingIncomeAfterSpending: BigDecimal = {
-    val totalIncome = maybeIncome.map(_.totalIncome).getOrElse(BigDecimal(1000))
-    val totalSpending = maybeSpending.map(_.totalSpending).getOrElse(BigDecimal(500))
+    val totalIncome = maybeIncome.map(_.totalIncome).getOrElse(throw new IllegalArgumentException("attempted to retrieve total income when there was no income"))
+    val totalSpending = maybeSpending.map(_.totalSpending).getOrElse(throw new IllegalArgumentException("attempted to retrieve total spending when there was no spending"))
     totalIncome - totalSpending
   }
 
-  //TODO OPS-9610 update this
   def requireScheduleIsDefined(): Unit = {
```

Comment:
Worth adding that `maybeIncome` and `maybeSpending` is defined? I don't think this method is actually necessary when it's called, I think it is only called when income and spending needs to be defined, but it seems that this code has these extra protections, so by adding `maybeIncome` and `maybeSpending` defined them we maintain the method